expyfun/tests/test_experiment_controller.py

```diff
@@ -197,6 +197,15 @@ def test_ec(ac=None, rd=None):
         assert_raises(ValueError, ec.get_presses)
         ec.listen_presses()
         assert_equal(ec.get_presses(), [])
+        assert_equal(ec.get_presses(type='presses'), [])
+        if this_rd == 'tdt':
+            # TDT does not have key release events, so should raise an
+            # exception if asked for them:
+            assert_raises(RuntimeError, ec.get_presses, type='releases')
+            assert_raises(RuntimeError, ec.get_presses, type='both')
+        else:
+            assert_equal(ec.get_presses(type='both'), [])
+            assert_equal(ec.get_presses(type='releases'), [])
```

Comment:
So hopefully eventually your changes should make it so that `assert_raises(ValueError, ec.get_presses, kind='foo')` will work